project/soulrrrrr/README.rst

```diff
@@ -0,0 +1,85 @@
+========================
+Columnar Array
+========================
+
+Basic Information
+=================
+
+https://github.com/soulrrrrr/ColumnarArray
+
+Problem to Solve
+================
+
+This problem is from https://yyc.solvcon.net/en/latest/nsd/schedule/22au_nycu/schedule.html#columnar-array.
```

Comment:
Please reword the problem by using your own description.

Reply:
I added my description of this problem.